<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Ashish" failures="2" tests="2" name="Test(failed)(failed)(failed)" time="8.453" errors="0" timestamp="2022-10-02T21:23:37 IST">
  <testcase classname="Sauce_Demo_TestClass.TC_04_Sauce_Demo_Verify_Products_ADDTOCART" name="Verify_SingleProduct_ADDTOCART" time="0.251">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]
at Sauce_Demo_TestClass.TC_04_Sauce_Demo_Verify_Products_ADDTOCART.Verify_SingleProduct_ADDTOCART(TC_04_Sauce_Demo_Verify_Products_ADDTOCART.java:19)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- Verify_SingleProduct_ADDTOCART -->
  <testcase classname="Sauce_Demo_TestClass.TC_01_Verify_Login_Functionality" name="INvalid_Credentials" time="0.0">
    <failure type="org.testng.TestNGException" message="
Cannot inject @Test annotated Method [INvalid_Credentials] with [class java.util.concurrent.TimeUnit].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection">
      <![CDATA[org.testng.TestNGException: 
Cannot inject @Test annotated Method [INvalid_Credentials] with [class java.util.concurrent.TimeUnit].
For more information on native dependency injection please refer to https://testng.org/doc/documentation-main.html#native-dependency-injection
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- INvalid_Credentials -->
  <testcase classname="Sauce_Demo_TestClass.TC_01_Verify_Login_Functionality" name="Open_SauceDemoLOgin" time="0.0">
    <ignored/>
  </testcase> <!-- Open_SauceDemoLOgin -->
</testsuite> <!-- Test(failed)(failed)(failed) -->
